src/Components/Contacts/ContactItem.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const ContactItem = () => {
+    return (
+        <div>
```

Comment:
If your component has no children the don't use opening and closing tags just write (<div />)